packages/stem-begroot/src/stem-begroot.tsx

```diff
@@ -278,6 +280,9 @@ function StemBegroot({
           currentStep={currentStep}
           steps={['Kies', 'Overzicht', 'Stemcode', 'Stem']}
         />
+    
+      <Spacer size={1} />
+
```

Comment:
In de andere componenten heb je de `Spacer` verwijderd, kun je die hier niet ook verwijderen?

Reply:
Kan inderdaad. Alleen moet het wel op een andere plek in de css weer opgevangen worden. Dit deed ik eerst met --nlds variable, maar dat is een beetje overkill. Dus had bedacht om het voor nu te laten.